tests/PayloadTest.php

```diff
@@ -86,12 +86,14 @@ public function testConvertToJSon()
             ->setThreadId('tread-id')
             ->setContentAvailability(true)
             ->setMutableContent(true)
-            ->setCustomValue('key', 'value');
-
+            ->setCustomValue('key', 'value')
+            ->setCustomValue('custom_array', $custom_values);
         $this->assertJsonStringEqualsJsonString(
             '{"aps": {"alert": {"title": "title"}, "badge": 1, "sound": "sound", "category": "category", ' .
-            ' "thread-id": "tread-id", "mutable-content": 1, "content-available": 1}, "key": "value"}',
+            ' "thread-id": "tread-id", "mutable-content": 1, "content-available": 1}, "key": "value", "custom_array": [1,2,3]}',
             $payload->toJson()
         );
+        $decoded_json = json_decode($payload->toJson());
+        $this->assertEquals(gettype($decoded_json->custom_array), 'array');
```

Comment:
Please, only one assert per test.
And I use camelCase for variables naming :)

Reply:
I should do another test just for 'custom_array' then ? :)

Reply:
Yes, please. I think we should have it to not reproduce this error in the future.